js/apps/admin-ui/src/components/scroll-form/FormPanel.tsx

```diff
@@ -21,8 +21,10 @@ export const FormPanel = ({
   scrollId,
   className,
 }: PropsWithChildren<FormPanelProps>) => {
+  const id = useId();
+
   return (
-    <Card className={className} isFlat>
+    <Card id={id} className={className} isFlat>
```

Comment:
Looks like PatternFly triggers some accessibility issues, adding a unique identifier here seems to resolve the issue.

Reply:
Oh, good catch @jonkoops